Application/global_settings_manager.py

```diff
@@ -68,6 +68,17 @@ def load_global_settings(self):
         settings = QSettings()
 
         settings.beginGroup("global-settings")
+
+        settings.beginGroup("user-settings")  # open user settings
+
+        # Sets the padding and the max width to global settings entity
+        self.global_settings_entity.table_padding = settings.value("table_padding")
```

Comment:
QSettings stores all values as strings, and thus we're setting the table padding, width, and cell size fields of the global settings instance to string values. These fields should be the int type. We'll want to do the conversion here. This comment applies to lines 75,76, 79, 80